### PR TITLE
Tissue bridges comparison

### PR DESCRIPTION
**Pull request by valosekj**
This PR adds scripts to compare tissue bridges obtained using different methods (manual, semi-automatic, automatic). More details in commit messages and [README](https://github.com/ivadomed/model_seg_sci/blob/jv/tissue_bridges_comparison/tissue_bridges/README.md).

### CODE REVIEW

File: tissue_bridges/README.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+## 0. Generate QC
+
+Generate QC to compare ground truth vs predicted lesions.
+
+### Ground truth
+```bash
+cd ~/data/data.neuro.polymtl.ca/sci-zurich
+for sub in sub-zh1[0-9][0-9];do sct_qc -i ${sub}/ses-01/anat/${sub}_ses-01_acq-sag_T2w.nii.gz -d derivatives/labels/${sub}/ses-01/anat/${sub}_ses-01_acq-sag_T2w_lesion-manual.nii.gz -s derivatives/labels/${sub}/ses-01/anat/${sub}_ses-01_acq-sag_T2w_seg-manual.nii.gz -p sct_deepseg_lesion -plane sagittal -qc-subject $sub -qc ~/data/results/sci-zurich/sci-zurich_2024-06-18_manualGT/qc;done
+```
+
```
Comment (naga-karthik) [inline tissue_bridges/README.md:18]: Add a note saying that the manual measurements were only provided for subjects `sub-zh101` to `sub-zh119`

File: tissue_bridges/README.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+single file.
+
+Example usage:
+
+```bash
+cd ~/code/sci-zurich/tissue_bridges
+
+python 02_read_xls_files.py
+-dir ~/data/results/sci-zurich/sci-zurich_2024-06-24_SCIsegV2_DA5_predictions/fold_0/results
+-o ~/data/results/sci-zurich/sci-zurich_2024-06-24_SCIsegV2_DA5_predictions/fold_0/stats
```
Comment (naga-karthik) [inline tissue_bridges/README.md:70]:
```suggestion
python 02_read_xls_files.py \
-dir ~/data/results/sci-zurich/sci-zurich_2024-06-24_SCIsegV2_DA5_predictions/fold_0/results \
-o ~/data/results/sci-zurich/sci-zurich_2024-06-24_SCIsegV2_DA5_predictions/fold_0/stats \
```

I thought multi-line python commands should always have a trailing `\` ? (if not, then please ignore the suggestion)

File: tissue_bridges/README.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+Compare tissue bridges obtained using three methods (manual, semi-automatic, automatic) and compute statistics.
+
+Example usage:
+
+```bash
+cd ~/code/sci-zurich/tissue_bridges
+
+python 03_compute_statistics.py
+-dir tissue_bridges.csv
```
Comment (naga-karthik) [inline tissue_bridges/README.md:84]:
```suggestion
python 03_compute_statistics.py \
-dir tissue_bridges.csv
```

**Review by naga-karthik — APPROVED** [review]
Maybe a note should be added in the README saying that the currently the tissue bridges method assumes that there's only 1 lesion (and multiple lesions is a work-in-progress?)

Apart from that, everything looks good to me!

File: tissue_bridges/README.md (hunk 1 above)
Comment (valosekj) [reply tissue_bridges/README.md:25]: good point! added in https://github.com/ivadomed/model_seg_sci/pull/88/commits/e2780a04a99c647bca417851de52fe98ac5bd91c

File: tissue_bridges/README.md (hunk 2 above)
Comment (valosekj) [reply tissue_bridges/README.md:77]: good catch! agree, thanks!

**Comment by valosekj** [comment]
> Maybe a note should be added in the README saying that the currently the tissue bridges method assumes that there's only 1 lesion (and multiple lesions is a work-in-progress?)

Good point! Added in https://github.com/ivadomed/model_seg_sci/pull/88/commits/01ed607391e012d95a93bc6f6102f14f83ed0027.

> Apart from that, everything looks good to me!

Okay, thank you! Merging!

**Pull request merged**